### PR TITLE
Formats upd

### PR DESCRIPTION
**Pull request by V1A0**
Replaced "{}".format(s) to f"{s}" where it's not critical for perception.

### CODE REVIEW

File: h8mail/utils/classes.py [hunk 1]
```diff
@@ -141,14 +139,14 @@ def get_hibp3(self, api_key):
...
-                    "No breaches found for {} using HIBP v3".format(self.target)
+                    f"No breaches found for {self.target} using HIBP v3"
                 )
+
             else:
                 c.bad_news(
-                    "HIBP v3: got API response code {code} for {target}".format(
-                        code=response.status_code, target=self.target
-                    )
+                    f"HIBP v3: got API response code {response.status_code} for {self.target}"
```
Comment (V1A0) [inline h8mail/utils/classes.py:147]: i guess it's even more accurate description

File: h8mail/utils/classes.py [hunk 2]
```diff
@@ -188,14 +185,13 @@ def get_hibp3_pastes(self):
...
                 c.info_news(
-                    "No pastes found for {} using HIBP v3 PASTE".format(self.target)
+                    f"No pastes found for {self.target} using HIBP v3 PASTE"
                 )
             else:
                 c.bad_news(
-                    "HIBP v3 PASTE: got API response code {code} for {target}".format(
-                        code=response.status_code, target=self.target
-                    )
+                    f"HIBP v3 PASTE: got API response code {response.status_code} for {self.target}"
```
Comment (V1A0) [inline h8mail/utils/classes.py:192]: i guess it's even more accurate description

File: h8mail/utils/classes.py [hunk 3]
```diff
@@ -319,7 +315,7 @@ def get_emailrepio(self, api_key=""):
                         self.data.append(
                             (
                                 "EMAILREP_LEAKS",
-                                "{} leaked credential".format(data["references"]),
+                                f"{data['references']} leaked credential",
```
Comment (V1A0) [inline h8mail/utils/classes.py:318]: I'm not sure it's a best way but it looks well

File: h8mail/utils/classes.py [hunk 4]
```diff
@@ -464,9 +460,7 @@ def get_hunterio_private(self, api_key):
         try:
             c.info_news("[" + self.target + "]>[hunter.io private]")
             target_domain = self.target.split("@")[1]
-            url = "https://api.hunter.io/v2/domain-search?domain={target}&api_key={key}".format(
-                target=target_domain, key=api_key
-            )
+            url = f"https://api.hunter.io/v2/domain-search?domain={target_domain}&api_key={api_key}"
```
Comment (V1A0) [inline h8mail/utils/classes.py:463]: Easy to read

File: h8mail/utils/classes.py [hunk 5]
```diff
@@ -446,7 +442,7 @@ def get_hunterio_public(self):
         try:
             c.info_news("[" + self.target + "]>[hunter.io public]")
             target_domain = self.target.split("@")[1]
-            url = "https://api.hunter.io/v2/email-count?domain={}".format(target_domain)
+            url = f"https://api.hunter.io/v2/email-count?domain={target_domain}"
```
Comment (V1A0) [inline h8mail/utils/classes.py:445]: Easy to read

File: h8mail/utils/localgzipsearch.py [hunk 6]
```diff
@@ -101,9 +101,7 @@ def local_search_single_gzip(files_to_parse, target_list):
         with gzip.open(file_to_parse, "r") as fp:
             size = os.stat(file_to_parse).st_size
             c.info_news(
-                "Searching for targets in {file_to_parse} ({size} bytes)".format(
-                    file_to_parse=file_to_parse, size=size
-                )
+                f"Searching for targets in {file_to_parse} ({size} bytes)"
```
Comment (V1A0) [inline h8mail/utils/localgzipsearch.py:104]: Same thing

File: h8mail/utils/localsearch.py [hunk 7]
```diff
@@ -130,9 +130,7 @@ def local_search_single(files_to_parse, target_list):
             size = os.stat(file_to_parse).st_size
             lines_no = raw_in_count(file_to_parse)
             c.info_news(
-                "Searching for targets in {file_to_parse} ({size} bytes, {lines_no} lines)".format(
-                    file_to_parse=file_to_parse, size=size, lines_no=lines_no
-                )
+                f"Searching for targets in {file_to_parse} ({size} bytes, {lines_no} lines)"
```
Comment (V1A0) [inline h8mail/utils/localsearch.py:133]: Same thing

File: h8mail/utils/breachcompilation.py [hunk 8]
```diff
@@ -32,7 +32,7 @@ def breachcomp_check(targets, breachcomp_path):
                         t.pwned += 1
                         t.data.append(("BC_PASS", re.split("[;:]",line)[-1]))
                         c.good_news(
-                            "Found BreachedCompilation entry {line}".format(line=line)
+                            f"Found BreachedCompilation entry {line}"
```
Comment (V1A0) [inline h8mail/utils/breachcompilation.py:35]: The same thing

File: h8mail/utils/classes.py [hunk 9]
```diff
@@ -25,10 +25,10 @@ def __init__(self, target_data, fp, ln, content):
...
 
     def dump(self):
-        print("Email: {}".format(self.target))
-        print("Path: {}".format(self.filepath))
-        print("Line: {}".format(self.line))
-        print("Content: {}".format(self.content))
+        print(f"Email: {self.target}")
+        print(f"Path: {self.filepath}")
+        print(f"Line: {self.line}")
+        print(f"Content: {self.content}")
```
Comment (V1A0) [inline h8mail/utils/classes.py:31]: The same thing

File: h8mail/utils/classes.py [hunk 10]
```diff
@@ -97,7 +97,7 @@ def make_request(
                 c.debug_news("DEBUG: RESPONSE HEADER---------------------")
                 print(
                     "\n".join(
-                        "{}: {}".format(k, v) for k, v in response.headers.items()
+                        f"{k}: {v}" for k, v in response.headers.items()
```
Comment (V1A0) [inline h8mail/utils/classes.py:100]: same

**Comment by khast3x** [comment]
Hello,
Thank you for the PR :purple_heart:.  
I'll be merging it the next release.  

Cheers!

File: h8mail/utils/classes.py (hunk 3 above)
Comment (khast3x) [reply h8mail/utils/classes.py:318]: I will manually test this one for safety

File: h8mail/utils/classes.py (hunk 4 above)
Comment (khast3x) [reply h8mail/utils/classes.py:463]: I will manually test this one

File: h8mail/utils/classes.py (hunk 3 above)
Comment (khast3x) [reply h8mail/utils/classes.py:318]: Looks good, I need to add error check for code 400 later

File: h8mail/utils/classes.py (hunk 4 above)
Comment (khast3x) [reply h8mail/utils/classes.py:463]: Looks good

**Pull request merged**